inbox/auth/microsoft.py

```diff
@@ -36,7 +36,7 @@ class MicrosoftAuthHandler(OAuthAuthHandler):
     )
     OAUTH_USER_INFO_URL = "https://outlook.office.com/api/v2.0/me"
 
-    OAUTH_SCOPE = " ".join(
+    OAUTH_AUTH_SCOPES = " ".join(
```

Comment:
nit: If it can be consistent, I would prefer to use "scope" := string, and "scopes" := list. That works at least with the OAUTH spec and our AuthAlligator API. Consistent naming of these constants isn't critical though, since editors and linting will identify them correctly.